doc/Doxyfile.in

```diff
@@ -125,7 +125,7 @@ EXAMPLE_PATTERNS       =
 EXAMPLE_RECURSIVE      = NO
 IMAGE_PATH             =
 INPUT_FILTER           =
-FILTER_PATTERNS        =
+FILTER_PATTERNS        = *.py="@FreeOrion_SOURCE_DIR@/doc/tag_filter.py"
```

Comment:
Thank you, this was indeed the issue.
I was able to move the starting quote and add escaped quotes for it to function.